content/news/2023-05-05-eosc-esg-openaire/index.md

```diff
@@ -8,6 +8,7 @@ supporters:
 - elixir
 - eosc
 - esg
+external_url: "https://www.openaire.eu/welcoming-an-open-science-webspace-for-the-galaxy-community"
```

Comment:
I believe this will make this entire article invisible, and it will directly link to this external URL, is this intentional?  After all the press release is linked from the bottom

Reply:
Right, thank you for the hint - this was in fact not intended... 🤦‍♂️ 
Maybe just reject the PR, as I found another typo in another news article accidentially some minutes ago. So I'll reshape the entire PR (MoU link stuff out, new typo correction in).

Reply:
Changes should be now in this updated PR